package.json

```diff
@@ -29,25 +30,40 @@
     "vue-multiselect": "^2.1.6",
     "vue-omnibox": "^0.3.7",
     "vue-sliding-pagination": "^1.3.2",
-    "vuedraggable": "^2.24.3",
-    "vuex": "^3.6.2"
+    "vuedraggable": "^2.24.3"
   },
   "devDependencies": {
-    "@babel/core": "^7.19.3",
+    "@babel/core": "^7.20.2",
+    "@babel/preset-env": "^7.20.2",
     "@storybook/addon-actions": "6.5.13",
     "@storybook/addon-docs": "6.5.13",
     "@storybook/addon-essentials": "6.5.12",
     "@storybook/addon-links": "6.5.12",
     "@storybook/vue": "6.5.12",
-    "babel-loader": "^8.2.5",
+    "@webpack-cli/generators": "^3.0.0",
+    "autoprefixer": "^10.4.13",
+    "babel-loader": "^9.1.0",
+    "css-loader": "^6.7.2",
     "js-beautify": "^1.14.7",
-    "vue-loader": "^15.9.8"
+    "mini-css-extract-plugin": "^2.7.0",
+    "postcss": "^8.4.19",
+    "postcss-loader": "^7.0.1",
+    "style-loader": "^3.3.1",
+    "vue": "2.7.14",
+    "vue-loader": "^15",
+    "vue-template-compiler": "^2.7.14",
+    "vuex": "^3.6.2",
+    "webpack": "^5.75.0",
+    "webpack-cli": "^5.0.0"
   },
   "scripts": {
     "build-css": "tailwindcss -i ./style/index.css -o ./style/style.css",
     "build-storybook": "build-storybook",
     "build-tokens": "node buildTokens.js",
-    "preinstall": "yarn build-tokens",
```

Comment:
instead of removing this hook, we could change it to "prepublish", to build the token files when publishing to npm.

Reply:
`prepublish` is the right place to run this.

Reply:
It should however be combined with the webpack build. Both have to run on `prepublish`.

Reply:
32798b1e148a01fdcfd089d49a19e39d028afe3d